docs/quickstart.md

```diff
@@ -47,61 +47,54 @@ You should get an output similar to this:
-.
-✔️  Created system_domain: 172.18.0.2.nip.io
+✔️  Created system_domain: 172.18.0.3.nip.io
 
 🚢 Deploying Workloads...
 ✔️  Workloads deployed
 
-🚢 Deploying Gitea.................................
-✔️  Gitea deployed (http://gitea.172.18.0.2.nip.io).
+🚢 Deploying Gitea......................................................
+✔️  Gitea deployed (http://gitea.172.18.0.3.nip.io).
 
-🚢 Deploying Registry..........
+🚢 Deploying Registry...............
 ✔️  Registry deployed
 
-🚢 Deploying Tekton.............................................................................
-✔️  Tekton deployed (http://tekton.172.18.0.2.nip.io).
+🚢 Deploying Tekton................................................................................
+✔️  Tekton deployed (http://tekton.172.18.0.3.nip.io).
 
-🚢 Deploying Core...........
-✔️  FuseML core component deployed (http://fuseml-core.172.18.0.2.nip.io).
+🚢 Deploying Core..............
+✔️  FuseML core component deployed (http://fuseml-core.172.18.0.3.nip.io).
 
 🚢 Downloading command line client...
-🚢 FuseML command line client saved as /home/jsuchome/kubernetes/fuseml/fuseml.
+🚢 FuseML command line client saved as /Users/flaviosr/workspace/fuseml/fuseml.
 Copy it to the location within your PATH (e.g. /usr/local/bin).
 
 🚢 To use the FuseML CLI, you must point it to the FuseML server URL, e.g.:
 
-    export FUSEML_SERVER_URL=http://fuseml-core.172.18.0.2.nip.io
+    export FUSEML_SERVER_URL=http://fuseml-core.172.18.0.3.nip.io
 
 ✔️  FuseML installed.
-System domain: 172.18.0.2.nip.io
+System domain: 172.18.0.3.nip.io
 ```
 
-The FuseML installer prints out URLs for _gitea_, _tekton_ and _fuseml-core_ components. You can access those sub-services directly at any given time. The fuseml core URL is especially important, as you'll need that to use the FuseML CLI.
+The FuseML installer prints out URLs for _gitea_, _tekton_ and _fuseml-core_ components. You can access those sub-services directly at any given time. The fuseml-core URL is especially important, as you'll need that to use the FuseML CLI.
 
 You can also see that the installer downloaded _fuseml_ command line client which you will use for actual FuseML operations. You can use the latest version which was downloaded automatically or manually choose one of [released versions](https://github.com/fuseml/fuseml-core/releases) from github.
 
-Use
+Run the following command to see the list of available commands:
 
 ```bash
 fuseml-installer install --help
 ```
 
-to get full information about the options the installer provides.
-
-### 4. Check if the components are up and running:
+## 4. Ensure FuseML components are up and running
 
 ```bash
-# Let's check the fuseml-core component
-kubectl get pods -n fuseml-core -o wide
-# Now let's check gitea
-kubectl get pods -n gitea -o wide
-# Let's check tekton
-kubectl get pods -n tekton-pipelines -o wide
+eval 'kubectl get pods -n '{istio-system,gitea,fuseml-registry,tekton-pipelines,fuseml-core}' -o wide;'
```

Comment:
Hm, while this is nice one-liner, I like the parts with more explanations more...

Reply:
ok. I can revert that change and include the missing services to be checked.

Reply:
done!

Reply:
thanks!